pages/national_resources/de_resources.md

```diff
@@ -18,6 +18,15 @@ national_resources:
       your_tasks: [dmp]
       your_domain: [plants]
     url: https://plan.nfdi4plants.org/
+  - name: ELN Finder
+    description: Electronic Laboratory Notebook (ELN) selection service. 
+    url: https://eln-finder.ulb.tu-darmstadt.de/home
```

Comment:
@wittigue maybe it would be useful to add some some related pages tags to these new tools?

Reply:
Perhaps tag for documentation and metadata page can be added/mentioned? I will approve and merge with current changes for now :)